package.json

```diff
@@ -21,7 +21,7 @@
     "deploy": "node scripts/deploy.js",
     "post-commit-status": "node scripts/postCommitStatus.js",
     "release": "pnpm prepare-publish && changeset publish && pnpm build:site && pnpm run deploy",
-    "prepare-publish": "pnpm build",
+    "prepare-publish": "pnpm build && tsx scripts/copyReadme.ts",
```

Comment:
My only nitpick is to name the file `copyReadme.cts` to signal that it's a CommonJS module. `__dirname` is not available in ESM but these loaders (`tsx` and co.) usually paper over it with shims.